jetty9-base/pom.xml

```diff
@@ -96,7 +112,7 @@
           </execution>
           <execution>
             <id>gcp-jars</id>
-            <phase>pre-integration-test</phase>
+            <phase>prepare-package</phase>
```

Comment:
jetty-base jar has to be built before these steps.
You can either move the maven-jar execution to prepare-package, or just change this to `package`.